book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
```

Comment:
- [ ]  You should create an entirely new instance of rentals here. It can go like Rental.new(date, book, self) but you should remember to add other parameters to the initializer of the rental class.